frontend/app/views/info/about.scala.html

```diff
@@ -100,7 +84,7 @@
     @* ===== Become a Patron ===== *@
     @for(img <- pageImages.find(_.name.contains("patrons"))) {
         <div class="page-slice l-constrained">
-            @fragments.promos.promoPrimary(title="Become a Patron and support the Guardian's future", img=img, toneClass=Some("tone-trans-brand-dark")) {
+            @fragments.promos.promoSecondary(title=Html("Become a Patron and support the Guardian's future"), img=img, toneClass=Some("tone-trans-brand-dark")) {
```

Comment:
It isn't, but I need to refactor the `promoSecondary` items to remove it in another PR